corehq/apps/reports/dispatcher.py

```diff
@@ -297,12 +297,13 @@ def dispatch(self, request, *args, **kwargs):
     def permissions_check(self, report, request, domain=None, is_navigation_check=False):
         from corehq.motech.repeaters.views import DomainForwardingRepeatRecords
         from corehq.apps.export.views.incremental import IncrementalExportLogView
+
         from corehq.toggles import INCREMENTAL_EXPORTS
 
         if (report.endswith(DomainForwardingRepeatRecords.__name__)
                 and not domain_has_privilege(domain, privileges.DATA_FORWARDING)):
             return False
-        if (report.endswith(IncrementalExportLogView.__name__) and not INCREMENTAL_EXPORTS.enabled(domain)):
+        if report.endswith(IncrementalExportLogView.__name__) and not INCREMENTAL_EXPORTS.enabled(domain):
```

Comment:
curious why this change made it into this PR? seems unrelated

Reply:
Ah I originally had an additional if statement here, and PyCharm peer pressured me into removing redundant parenthesis. I'll put it back just to avoid confusion